### PR TITLE
James/alliance demo tickets

### PR DESCRIPTION
**Pull request by james-vendia**
Create alliance partner ticket sharing demo

### CODE REVIEW

File: demos/travel-and-hospitality/alliance-ticket-sharing/resources/american-tickets.gql [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+            ],
+            fees: [
+                {
+                    code: XF,
+                    value: {
+                        amount: 11.20,
+                        currency: "USD",
+                        precision: 4
+                    }
+                }, {
```
Comment (brian-vendia) [inline demos/travel-and-hospitality/alliance-ticket-sharing/resources/american-tickets.gql:249]:
```suggestion
                },
                {
```

File: demos/travel-and-hospitality/alliance-ticket-sharing/resources/american-tickets.gql [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+                        amount: 9.00,
+                        currency: "USD",
+                        precision: 3
+                    }
+                }
+            ]
+        },
+        aclInput: {
+            acl: [
+                { principal: {nodes: ["AmericanNode"]}, operations: [ALL, UPDATE_ACL] },
```
Comment (brian-vendia) [inline demos/travel-and-hospitality/alliance-ticket-sharing/resources/american-tickets.gql:261]: I think this is the implicit behavior. Calling this out explicitly for the sake of the demo?

File: demos/travel-and-hospitality/alliance-ticket-sharing/resources/iberia-tickets.gql [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+                        amount: 9.00,
+                        currency: "EUR",
+                        precision: 3
+                    }
+                }
+            ]
+        },
+        aclInput: {
+            acl: [
+                { principal: {nodes: ["IberiaNode"]}, operations: [ALL, UPDATE_ACL] },
```
Comment (brian-vendia) [inline demos/travel-and-hospitality/alliance-ticket-sharing/resources/iberia-tickets.gql:262]: I think this is the implicit behavior. Calling this out explicitly for the sake of the demo?

File: demos/travel-and-hospitality/alliance-ticket-sharing/uni_configuration/schema.json [hunk 4]
```diff
@@ -0,0 +1,558 @@
...
+        "properties": {
+          "passengerNameRecordCode": {
+            "type": "string",
+            "description": "A unique passenger record, represented by a 6-digit alphanumeric value",
+            "pattern": "[A-Z0-9]{6}"
+          },
+          "airlineCode": {
+            "type": "string",
+            "description": "The 2-letter code of the issuing airline",
+            "pattern": "[A-Z]{2}"
```
Comment (brian-vendia) [inline demos/travel-and-hospitality/alliance-ticket-sharing/uni_configuration/schema.json:26]: I know this is a demo with known inputs, but this character class will only allow 2 upper case letters. Two lower case letters will fail validation.

File: demos/travel-and-hospitality/alliance-ticket-sharing/uni_configuration/schema.json [hunk 5]
```diff
@@ -0,0 +1,558 @@
...
+                "format": "date"
+              },
+              "frequentFlyerInformation": {
+                "type": "object",
+                "description": "Passenger's frequent flyer information",
+                "properties": {
+                  "airlineCode": {
+                    "type": "string",
+                    "description": "The 2-letter code of the airline",
+                    "pattern": "[A-Z]{2}"
```
Comment (brian-vendia) [inline demos/travel-and-hospitality/alliance-ticket-sharing/uni_configuration/schema.json:70]: Same question about the character class only allowing upper case letters.

File: demos/travel-and-hospitality/alliance-ticket-sharing/resources/american-tickets.gql (hunk 2 above)
Comment (james-vendia) [reply demos/travel-and-hospitality/alliance-ticket-sharing/resources/american-tickets.gql:261]: Yes, exactly.  Trying to make the implicit explicit to show the full range of ACL behavior.

File: demos/travel-and-hospitality/alliance-ticket-sharing/uni_configuration/schema.json (hunk 5 above)
Comment (james-vendia) [reply demos/travel-and-hospitality/alliance-ticket-sharing/uni_configuration/schema.json:70]: Intentional

File: demos/travel-and-hospitality/alliance-ticket-sharing/resources/american-tickets.gql [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+                    }
+                }
+            ],
+            prices: {
+                baseFareAmount: {
+                    amount: 375.00,
+                    currency: "USD",
+                    precision: 5
+                }
+            }, taxes: [
```
[the remark this answers is not in the document]
Comment (james-vendia) [reply demos/travel-and-hospitality/alliance-ticket-sharing/resources/american-tickets.gql:95]: Not sure why my code editor is not formatting properly.  I'll adjust.

File: demos/travel-and-hospitality/alliance-ticket-sharing/resources/american-tickets.gql [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+            ],
+            fees: [
+                {
+                    code: XF,
+                    value: {
+                        amount: 11.20,
+                        currency: "USD",
+                        precision: 4
+                    }
+                }, {
```
[the remark this answers is not in the document]
Comment (james-vendia) [reply demos/travel-and-hospitality/alliance-ticket-sharing/resources/american-tickets.gql:113]: Not sure why my code editor is not formatting properly.  I'll adjust.

**Pull request merged**